modules/gltf/src/index.ts

```diff
@@ -0,0 +1,10 @@
+// luma.gl, MIT license
+
+export type {PBREnvironment} from './pbr/pbr-environment';
+export type {ParsePBRMaterialOptions, ParsedPBRMaterial} from './pbr/parse-pbr-material';
+export {parsePBRMaterial} from './pbr/parse-pbr-material';
+
+// glTF Scenegraph Instantiator
+export {createScenegraphsFromGLTF, createScenegraphsFromGLTF as createGLTFObjects} from './gltf/create-gltf-objects';
```

Comment:
Since we are breaking a lot (including the module name), I think we can remove such re-exports. A minimal bullet in the upgrade-guide should be enough.